lib/init.coffee

```diff
@@ -11,4 +15,44 @@ module.exports =
         'Available codes: https://pypi.python.org/pypi/pep257#error-codes')
 
   activate: ->
-    console.log 'activate linter-pep257'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-pep257.executablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+    @subscriptions.add atom.config.observe 'linter-pep257.ignoreCodes',
+      (ignoreCodes) =>
+        @ignoreCodes = ignoreCodes
+
+  deactivate: ->
+    @subscriptions.dispose()
+
+  lintPath: (params) ->
+    return helpers.exec(@executablePath, params, {stream: 'stderr'})
+
+  parseMessages: (output) ->
+    # parse lint output
+    messages = helpers.parse(output, "^(?<file>.+):(?<line>\\d+)\\s+(?<message>.+)")
+    output = output.split(/\r?\n/)
+    messages.forEach (msg, i) ->
+      msg.type = "Info"
+      text = (output[i+i+1] || "").trim()  # get the second line of each message
```

Comment:
The second line of pep257 is indeed the important one. What might be better here, instead of constructing an eyebrow raising index key, conform the pep257 output to online before sending it to the helper, something like:

```js
output = output.replace(/:$\s+/mg, ' ')
messages = helpers.parse()
```